examples/health_check/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/goflash/flash/v2"
+)
+
+// main demonstrates the health check functionality in GoFlash.
+func main() {
+	app := flash.New()
+
+	// Enable health check at /health
+	app.EnableHealthCheck("/health")
```

Comment:
It would be nice to have HealthCheckConfig (optional) where you would provide path and additional params if any. Also there can be "HealthCheckFunc", "OnErrorFunc", "OnSuccessFunc".